server/keycloak.js

```diff
@@ -260,6 +260,11 @@ class Keycloak {
     }
   }
 
+  getUserUrl(userId = '') {
+    if (!userId) throw new Error('keycloak: User ID is required');
```

Comment:
If `userId` defaults to `''`, doesn't it mean that this condition will always be false?

Reply:
No. Defaulted means here if no arg is supplied it will take userId as an empty string and throw an error ...